x-pack/plugins/security_solution/public/timelines/components/open_timeline/helpers.ts

```diff
@@ -254,13 +251,21 @@ export const getTimelineStatus = (
 export const defaultTimelineToTimelineModel = (
   timeline: TimelineResult,
   duplicate: boolean,
-  timelineType?: TimelineType
+  timelineType?: TimelineType,
+  unifiedComponentsInTimelineEnabled?: boolean
```

Comment:
The current approach forces all the consumers of this function to retrieve this value using `useIsExperimentalFeatureEnabled` and pass.

Wouldn't it be easier to use `ExperimentalFeaturesService` singleton inside this file, instead of receiving this value by parameter? It would simplify the cleaning when we remove this flag too.

Reply:
@semd , I did not even know that it existed. I agree with you but I have three more upcoming PRs for this functionality. Do you mind, if I do that change in next PR ?

Reply:
Yes sure, it's just a simplification, and also to reduce the scope of the changes. But if you keep it this way it's also fine, as you prefer 👍

Reply:
Actually I will have to check the Singleton class API but I will make changes accordingly in the next PR. 

I would be glad as well if I could remove this extra param from everywhere.